cea/default.config

```diff
@@ -52,6 +52,42 @@ archetypes.type = MultiChoiceParameter
 archetypes.choices = comfort, architecture, HVAC, internal-loads, supply, restrictions
 archetypes.help = List of input databases to create.
 
+[streets-helper]
+bbox =
+bbox.type = ListParameter
+bbox.help = Default: empty, it uses district shapefile as bounding box, otherwise indicate Bounding box coordinates. bbox = min Longitude , min Latitude , max Longitude , max Latitude, if empty we
+
+streets = all_private
+streets.type = ChoiceParameter
```

Comment:
I wonder if a MultiChoiceParameter would be more suitabe?

Reply:
not for the moment. It is supposed to be only one choice for now.